_facultyPages/stud_mgmt_pg.php

```diff
@@ -0,0 +1 @@
+Magic
```

Comment:
Why does this exist?

Reply:
To test the link... I have since built the page. It will be in the next round.